src/components/Spot/Spot.spec.js

```diff
@@ -87,21 +87,179 @@ describe('Single Spot-Display', () => {
 		const spotDetail4 = screen.queryByText(/-78.268353/i);
 		const spotDetail5 = screen.queryByText(/nice and warm/i);
 
-		const nameAnimal1 = screen.queryByText(/Dolphin/i);
-		const nameAnimal2 = screen.queryByText(/Shark/i);
-
-		const nameOrganizer1 = screen.queryByText(/Unexso/i);
-		const nameOrganizer2 = screen.queryByText(/Silversea Cruises/i);
-		const nameOrganizer3 = screen.queryByText(/Pacific Fleet/i);
-
 		expect(spotDetail1).toBeInTheDocument();
 		expect(spotDetail2).toBeInTheDocument();
 		expect(spotDetail3).toBeInTheDocument();
 		expect(spotDetail4).toBeInTheDocument();
 		expect(spotDetail5).toBeInTheDocument();
+	});
+
+	it('renders the names of animals living there & NOT the names of animals not living there', () => {
+		render(
+			<Spot
+				spot={{
+					id: '2',
+					name: 'Gran Bahama',
+					country: 'Bahamas',
+					geoTag: { latitude: '26.661393', longitude: '-78.268353' },
+					description: 'nice and warm',
+				}}
+				animals={[
+					{
+						id: '1',
+						name: 'Dolphin',
+						spots: ['Gran Bahama', 'Bimini', 'Galápagos'],
+						description: 'friendly, very intelligent, likes to play',
+					},
+					{
+						id: '2',
+						name: 'Shark',
+						spots: ['Galápagos', 'Socorro Islands'],
+						description:
+							'despite of persistent rumors, sharks are NOT aggressiv, but maybe a bit hungry from time to time..',
+					},
+				]}
+				organizers={[
+					{
+						id: '1',
+						name: 'Unexso',
+						spots: [
+							{
+								name: 'Freeport',
+								area: 'Gran Bahama',
+								animals: ['Dolphin'],
+							},
+						],
+					},
+					{
+						id: '2',
+						name: 'Silversea Cruises',
+						spots: [
+							{
+								name: 'San Cristóbal',
+								area: 'Galápagos',
+								animals: ['Dolphin', 'Shark'],
+							},
+							{
+								name: 'Socorro Islands',
+								area: 'Colima, Revillagigedo Archipelago',
+								animals: ['Shark, Dolphin'],
+							},
+						],
+					},
+					{
+						id: '3',
+						name: 'Pacific Fleet',
+						spots: [
+							{
+								name: 'Socorro Islands',
+								area: 'Colima, Revillagigedo Archipelago',
+								animals: ['Shark, Dolphin'],
+							},
+							{
+								name: 'Neptun Islands',
+								area: 'South Australia, Spencer Gulf',
+								animals: 'Shark',
+							},
+							{
+								name: 'Freeport',
+								area: 'Gran Bahama',
+								animals: ['Dolphin'],
+							},
+						],
+					},
+				]}
+			/>
+		);
+
+		const nameAnimal1 = screen.queryByText(/Dolphin/i);
+		const nameAnimal2 = screen.queryByText(/Shark/i);
 
 		expect(nameAnimal1).toBeInTheDocument();
 		expect(nameAnimal2).not.toBeInTheDocument();
+	});
+
+	it('renders the names of organizers at this location & NOT the names of organizers not located at this location', () => {
```

Comment:
Could you rephrase like
```suggestion
	it('renders only the names of organizers at this location', () => {
```
?